src/tools/roc_send/main.cpp

```diff
@@ -101,25 +107,38 @@ int main(int argc, char** argv) {
     sndio::Config io_config;
     io_config.sample_spec.set_channel_set(sender_config.input_sample_spec.channel_set());
 
-    if (args.packet_length_given) {
-        if (!core::parse_duration(args.packet_length_arg, sender_config.packet_length)) {
-            roc_log(LogError, "invalid --packet-length");
+    if (args.packet_len_given) {
+        if (!core::parse_duration(args.packet_len_arg, sender_config.packet_length)) {
+            roc_log(LogError, "invalid --packet-len");
             return 1;
         }
     }
 
-    if (args.frame_length_given) {
-        if (!core::parse_duration(args.frame_length_arg, io_config.frame_length)) {
-            roc_log(LogError, "invalid --frame-length: bad format");
+    if (!args.max_packet_size_given) {
+        // 2 bytes per sample?
```

Comment:
Please add this comment:

```
// TODO(gh-608): take size from --packet-encoding instead of assuming 2 bytes per sample
```

In https://github.com/roc-streaming/roc-toolkit/issues/608, we'll add --packet-encoding option that will allow to select which format to use. Currently we always use 16-bit PCM, so it's OK to keep it `2` until 608 is implemented.